runtime/gc_glue_java/ObjectModelDelegate.hpp

```diff
@@ -461,19 +461,6 @@ class GC_ObjectModelDelegate
 	void calculateObjectDetailsForCopy(MM_EnvironmentBase *env, MM_ForwardedHeader *forwardedHeader, uintptr_t *objectCopySizeInBytes, uintptr_t *objectReserveSizeInBytes, uintptr_t *hotFieldAlignmentDescriptor);
 #endif /* defined(OMR_GC_MODRON_SCAVENGER) */
 
-	/**
-	 * Calculate the actual object size and the size adjusted to object alignment. The calculated object size
-	 * includes any expansion bytes allocated if the object will grow when moved. TODO: Merge this with the
-	 * above calculateObjectDetailsForCopy()
-	 *
-	 * @param[in] env points to the environment for the calling thread
-	 * @param[in] forwardedHeader pointer to the MM_ForwardedHeader instance encapsulating the object
-	 * @param[out] objectCopySizeInBytes actual object size
-	 * @param[out] objectReserveSizeInBytes size adjusted to object alignment
-	 * @param[out] doesObjectNeedHash flag that indicates if object needs hashing
-	 */
-	void calculateObjectDetailsForCopy(MM_EnvironmentBase *env, MM_ForwardedHeader* forwardedHeader, uintptr_t *objectCopySizeInBytes, uintptr_t *objectReserveSizeInBytes, bool *doesObjectNeedHash);
```

Comment:
There is a mutual dependency between Delegate/OpenJ9 variant and OMR variant of this method.
Let's deal with it like this:
1) keep this method (could be inlined and empty with comment 'to be removed'), since CopyForward will not rely on it, and we deliver this change first
2) deliver OMR change that will remove the only user of this method
3) remove this method with another OpenJ9 change